wrangler-core/src/main/java/io/cdap/directives/column/ChangeColCaseNames.java

```diff
@@ -94,5 +104,40 @@ public Mutation lineage() {
       .all(Many.of())
       .build();
   }
+  @Override
+  public Relation transform(RelationalTranformContext relationalTranformContext,
+                            Relation relation) {
+    java.util.Optional<ExpressionFactory<String>> expressionFactory = getExpressionFactory(relationalTranformContext);
+    if (!expressionFactory.isPresent()) {
+      return new InvalidRelation("Cannot find an Expression Factory");
+    }
+    List<String> columnNames = generateListCols(relationalTranformContext);
+    Map<String, Expression> colmap = generateColumnCaseMap(columnNames, expressionFactory.get());
+    return relation.select(colmap);
+  }
+
+  private List<String> generateListCols(RelationalTranformContext relationalTranformContext) {
+    List<String> colnames = new ArrayList<String>();
+    java.util.Set<String> s = relationalTranformContext.getInputRelationNames();
+    for (String inp : s) {
+      Schema schema = relationalTranformContext.getInputSchema(inp);
```

Comment:
Right now this is not an issue as we only support executing one directive in SQL. But later on if we add multiple directives support, input schema may not be the correct thing to use as columns may have been dropped/added/renamed in previous steps. Will the input schema passed have the previous directive's transforms applied in that case?